funding/manager.go

```diff
@@ -2530,6 +2730,20 @@ func (f *Manager) sendFundingLocked(
 	}
 	fundingLockedMsg := lnwire.NewFundingLocked(chanID, nextRevocation)
 
+	// We'll include the alias ShortChannelID for regular option_scid_alias
+	// channels and for zero-conf channels. The TLV segment will not be
+	// included for non-option_scid_alias channels.
+	if completeChan.ChanType.IsZeroConf() {
+		// The alias will be stored under ShortChannelID.
+		aliasScid := completeChan.ShortChanID()
```

Comment:
We'll also need similar logic in link.go, feel fee to insta resolve if it's there, as still getting through the PR.

Reply:
yup its there